app/scripts/modules/core/src/domain/ITrigger.ts

```diff
@@ -37,6 +37,7 @@ export interface ICronTrigger extends ITrigger {
 export interface IPubsubTrigger extends ITrigger {
   pubsubSystem: string;
   subscriptionName: string;
+  constraints: any;
```

Comment:
Does `map-editor` expect a `Map` here? (If that's even a TS type)

Reply:
It's probably completely untyped, but if anything it would expect `{[key: string]: string}`.